example-streamer-uses/Cargo.toml

```diff
@@ -21,27 +21,40 @@ license.workspace = true
 
 [[bin]]
 name = "me_client"
+required-features = ["enable-vsomeip-transport"]
 
 [[bin]]
 name = "me_publisher"
+required-features = ["enable-vsomeip-transport"]
 
 [[bin]]
 name = "me_service"
+required-features = ["enable-vsomeip-transport"]
 
 [[bin]]
 name = "me_subscriber"
+required-features = ["enable-vsomeip-transport"]
 
 [[bin]]
 name = "ue_client"
+required-features = ["enable-zenoh-transport"]
 
 [[bin]]
 name = "ue_publisher"
+required-features = ["enable-zenoh-transport"]
 
 [[bin]]
 name = "ue_service"
+required-features = ["enable-zenoh-transport"]
 
 [[bin]]
 name = "ue_subscriber"
+required-features = ["enable-zenoh-transport"]
+
+[features]
+enable-zenoh-transport = ["up-transport-zenoh", "zenoh"]
+enable-vsomeip-transport = ["up-transport-vsomeip"]
```

Comment:
How about just calling these `zenoh-transport` and `vsomeip-transport`? A feature needs to be enabled (by supplying the `cargo --features` parameter), but the feature itself should just be named according to the functionality it provides FMPOV ...

Reply:
Okay by me.